### PR TITLE
Improve lyrics

### PR DESCRIPTION
**Pull request by Sv443**
- [x] Add geniURL access token
- [x] Fetch lyrics lazily, without blocking other code exec
  
Unrelated stuff:  
- [x] Make /joke responses a bit prettier

### CODE REVIEW

File: src/commands/music/Play.ts [hunk 1]
```diff
@@ -114,6 +114,7 @@ export class Play extends Command {
         if(( res.loadType == "PLAYLIST_LOADED" ? reduceSongsLength(res.tracks) : res.tracks[0].duration ) + (player.queue.totalSize ?? 0) > four_hours 
             && !(await getPremium(guild.id)))
             return this.editReply(int, embedify("Total queue time will be over 4 hours, please upgrade to our premium tier to add more songs to your queue."));
+            // return this.editReply(int, embedify("Total queue time will be over 4 hours, please [upgrade to our premium tier](https://brewbot.co/premium) to add more songs to your queue."));
```
Comment (IllusionMan1212) [inline src/commands/music/Play.ts:117]: h
the entire music stuff is dead (for now)

File: src/commands/music/global.music.ts [hunk 2]
```diff
@@ -34,14 +34,14 @@ export const skipVotes: Record<string, ISkip> = {};
...
- * Using separate song and artist props is recommended so geniURL can optimize its results.
- */
+/** Fetches song info by song and artist name or query. Now with server-side filtering for both! */
 export async function fetchSongInfo(query: string | { song: string, artist: string }): Promise<SongInfo | undefined>
 {
+    /** geniURL fuzzy search threshold, lower = more strict, API default = 0.65 */
+    const threshold = 0.3;
     try
     {
+        /** Trim out stuff like `(Bunger Remix)` or `ft. DJ Bunger` */
```
Comment (IllusionMan1212) [inline src/commands/music/global.music.ts:44]: i love DJ Bunger

File: src/commands/music/Play.ts (hunk 1 above)
Comment (Sv443) [reply src/commands/music/Play.ts:117]: ik, but I already did this before it went down

**Review by IllusionMan1212 — APPROVED** [review]
gioerjohigejiwo :+1:

**Comment by Sv443** [comment]
george

**Pull request merged**